packages/aquedux-client/src/network/client.js

```diff
@@ -12,13 +12,8 @@ import * as eventHub from '../utils/eventHub'
 import localStorage from '../utils/localStorage'
 import onAquedux from '../utils/actionWrapper'
 
-const createAqueduxClient = (store, why, config) => {
-  store.dispatch(
-    actions.config.set({
-      pingAwareActionTypes: config.pingAwareActionTypes || {},
-      timeout: config.timeout || 5000
-    })
-  )
+const createAqueduxClient = (store, why, options) => {
```

Comment:
I thought the "why" argument was removed previously, it should be now (not used anymore).